core/src/main/java/com/sequenceiq/cloudbreak/service/rdsconfig/QueryProcessorRdsConfigProvider.java

```diff
@@ -0,0 +1,60 @@
+package com.sequenceiq.cloudbreak.service.rdsconfig;
```

Comment:
any unit test?

Reply:
@Amlesh1902 you can find few example files with running the following command from the project root:
`find ./ -name "*Test.java"`